website/docs/docs.md

```diff
@@ -256,6 +257,26 @@ module.exports = {
 };
 ```
 
+#### Toggling category open by default
+
+For sites that have collapsible categories, you may want more fine grain control over certain categories. If you want specific categories to be alwasy expanded, you can set `collapsed` to `false`:
+
+```js title="sidebars.js"
+module.exports = {
+  docs: {
+    Guides: [
+      'creating-pages',
+      {
+        type: 'category',
+        label: 'Docs',
+        collapsed: false,
+        items: ['markdown-features', 'sidebar', 'versioning'],
+      },
+    ],
+  },
+};
+```
+
```

Comment:
@yangshun I updated the docs - open to suggestions! I wasn't sure if I needed to update any other areas of the docs. Let me know!

Reply:
This looks fine. What's the current default value for the `collapsed` field? The default should be `true` to preserve backward compatibility.